server/utils/db/db_utils.py

```
@@ -1656,6 +1657,22 @@ def initialize_tables():
 
+            # Add shared_with_org flag — connections are private by default,
+            # only visible to org members when owner explicitly shares.
+            try:
+                cursor.execute(
+                    "ALTER TABLE user_connections ADD COLUMN IF NOT EXISTS shared_with_org BOOLEAN NOT NULL DEFAULT FALSE;"
+                )
+                conn.commit()
+                logging.info(
+                    "Ensured shared_with_org column exists on user_connections table."
+                )
+            except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)